scripts/irods/test/test_iadmin.py

```diff
@@ -1940,7 +1940,7 @@ def test_input_parameters_invalid(self):
     @unittest.skipUnless(plugin_name == 'irods_rule_engine_plugin-irods_rule_language', 'only applicable for irods_rule_language REP')
     def test_modifying_restricted_columns(self):
         dumb_string = 'garbage'
-        dumb_int = 16
+        dumb_int = '00000'
```

Comment:
@alanking Any thoughts?

I figured this value would never produce incorrect results.

Reply:
If this works, this is definitely better. Been meaning to fix this for a while. Occasionally, this test will fail because "16" appears in an ID (because, y'know, incremental).